packages/core/lib/config.ts

```diff
@@ -1,6 +1,16 @@
 import { type Plugin } from './plugin'
 import { isLogger, isNumber, isObject, isPluginArray, isString, isStringArray, isStringWithLength } from './validation'
 
+interface BugsnagErrorEvent {
+  setTraceCorrelation: (traceId: string, spanId: string) => void
```

Comment:
I'd suggest marking this optional as it could be missing

```suggestion
  setTraceCorrelation?: (traceId: string, spanId: string) => void
```